### PR TITLE
cli_rm_builder: Extract EXAMPLES and make it persistent on rerun

### LINKED ISSUE #2: Make `EXAMPLES` section in module file persistent on builder re-run
##### SUMMARY
If the module file has examples and the builder is re-run to rebuild the argspec (from either external docstring or module docstring), all of those are wiped out. This is an erroneous behavior and needs to be accounted for. As an user, I'd expect the `EXAMPLES` section to be preserved and maintained across argspec rebuilds. 

Use Cases:
- When module file doesn't exist and builder is run **with** external docstring having examples, the module file generated should have its `EXAMPLES` section populated with the given examples.
- When module file exists and the builder is run **without** an external docstring (i.e., module file being argspec SoT), if there are existing examples, those should be preserved.
- When module file exists and the builder is run **with** external docstring having examples, if there are existing examples, those should be overridden by docstring examples (same way existing argspec is overridden by external docstring argspec in such cases).

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
possibly get_doc.py (?)

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

Fixes #2

### CODE REVIEW

**Comment by GomathiselviS** [comment]
cc @NilashishC

File: roles/init/tasks/main.yml [hunk 1]
```diff
@@ -24,6 +24,15 @@
     rm_documentation: "{{ lookup('file', '{{ docstring }}') }}"
   when: docstring is defined
 
+- name: Extract EXAMPLES from Resource Module (for existing module)
+  set_fact:
+    example_documentation: "{{ lookup('file', '{{ path_to_module }}')|ansible_network.cli_rm_builder.get_example }}"
```
Comment (NilashishC) [inline roles/init/tasks/main.yml:29]: IIUC, with this, the examples in module file will get precedence over externally provided examples. We have followed the opposite for docstring.

Comment (NilashishC) [reply roles/init/tasks/main.yml:29]: I feel we should move this under the `block-when`. Wdyt?

Comment (GomathiselviS) [reply roles/init/tasks/main.yml:29]: yes. So task at line 27 extracts the example from the existing module file. Only when the result is empty, the example from the model location will be taken.

File: README.md [hunk 2]
```diff
@@ -51,6 +52,7 @@ ansible-playbook -e rm_dest=<destination for modules and module utils> \
 - `collection_name`: The name of the collection
 - `resource`: The network resource targeted by the module
 - `docstring`: The path to the file that contains docstring (in YAML)
+- `example`: The path to the file that contains examples (in txt)
```
Comment (NilashishC) [inline README.md:55]:
```suggestion
- `examples`: The path to the file that contains examples (in txt)
```

**Review by NilashishC — APPROVED** [review]
@GomathiselviS Thanks a lot for adding this!

**Pull request merged**